### ISSUE TITLE
Title: OCR Predictor Not Detecting Text Orientation

### ISSUE THREAD

**Issue by 1498akshay** [mindee/doctr#1913]
### Bug description

Expected Behavior:
The OCR predictor should return the detected text along with the correct orientation information.

Observed Behavior:
The text is extracted successfully, but no orientation details are included in the output.

Steps to Reproduce:
Initialize the ocr_predictor with detect_orientation=True.

Pass an image with rotated text.

Check if the output includes orientation details

The OCR predictor straightens the image but sometimes flips it upside down or rotates it incorrectly.

### Code snippet to reproduce the bug

predictor = ocr_predictor(
    det_arch="fast_base",
    reco_arch="crnn_mobilenet_v3_large",
    pretrained=True,
    assume_straight_pages=False,
    straighten_pages=True,
    preserve_aspect_ratio=False,
    detect_orientation=True
)

### Error traceback

'''no error'''
The OCR predictor should return the detected text along with the correct orientation information.
The OCR predictor straightens the image but sometimes flips it upside down or rotates it incorrectly.

### Environment

The OCR predictor should return the detected text along with the correct orientation information.
The OCR predictor straightens the image but sometimes flips it upside down or rotates it incorrectly.

### Deep Learning backend

is_tf_available: False
is_torch_available: True

**Comment by felixdittrich92** [comment]
Hi @1498akshay 👋,

Thanks for reporting.

I'm not able to reproduce the bug:

```python
from doctr.io import DocumentFile
from doctr.models import ocr_predictor

# ~90 degree rotated image
doc = DocumentFile.from_images("/home/felix/Desktop/doctr_test_data/106.jpeg")

predictor = ocr_predictor(
    det_arch="fast_base",
    reco_arch="crnn_mobilenet_v3_large",
    pretrained=True,
    assume_straight_pages=False,
    straighten_pages=True,
    preserve_aspect_ratio=False,
    detect_orientation=True
)

res = predictor(doc)
export = res.export()

for p in export['pages']:
    print(p['orientation'])

res.show()
```

result:
```
{'value': 88, 'confidence': None}
```

Would it be possible to share an example that we can reproduce the issue ?

The general page orientation can fail that's true we will retrain it at some point - but you can also fine tune it easily to your own needs - only a folder with non rotated images is required we will auto augment the images in the training script and generate the required labels for it.
ref.: https://mindee.github.io/doctr/using_doctr/custom_models_training.html#custom-orientation-classification-models

Best,
Felix

**Comment by felixdittrich92** [comment]
Additional:
If all the text on your samples is horizontal I would suggest to disable the crop orientation classifier.
If you have to only with small rotations ~-45 - 45 degree you could also disable the page orientation classifier
See: https://mindee.github.io/doctr/using_doctr/using_models.html#advanced-options
